paddle/fluid/framework/distributed_strategy.proto

```diff
@@ -43,6 +43,12 @@ message ShardingConfig {
   optional int32 pp_degree = 11 [ default = 1 ];
 }
 
+message HybridConfig {
+  optional int32 num_data_parallel = 1 [ default = -1 ];
+  optional int32 num_model_parallel = 2 [ default = 1 ];
+  optional int32 num_pipeline_parallel = 3 [ default = 1 ];
```

Comment:
Use dp_degree, mp_degree and pp_degree to be consistent with static graph.

Reply:
done